testsuite/features/secondary/min_ssh_push.feature

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Register a salt system to be managed via SSH push
+
+  Scenario: Delete the salt minion for ssh-reverse bootstrap
+    Given I am on the Systems overview page of this "sle_minion"
+    When I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    And I wait until I see "has been deleted" text
+    And I cleanup minion "sle_minion"
+    Then "sle_minion" should not be registered
+
+  Scenario: Register this minion for SSH push via tunnel
```

Comment:
for push via ssh tunnel maybe?

Reply:
via SSH tunnel then (capital, it's the protocol)